aws/lambda/python/reports_notification.py

```
@@ -6,8 +6,10 @@
 sns_topic_arn = os.environ["SNS_TOPIC_ARN"]
 client = boto3.client('sns')
 
+
 def generate_build_succeeding_message(tests):
```

Ruff

Missing return type annotation for public function `generate_build_succeeding_message`

Add return type annotation: `str`

(ANN201)